### ISSUE TITLE
enhancement: text substitutions in macOS

### ISSUE THREAD

**Issue by boneskull** [TryGhost/Ghost-Desktop#304]
I'd like to be able to use text substitutions (specifically, "smart quotes" and "smart dashes") in `Ghost.app`.

Here's an example screenshot from `Bear.app`:

![image](https://user-images.githubusercontent.com/924465/33502908-b930abfa-d696-11e7-9f48-21a2fa8571fd.png)

Likewise, there's this module which *could* provide what I want: https://npm.im/electron-text-substitutions

Is there any interest in this?  If so, I could attempt a PR.

**Comment by felixrieseberg** [comment]
Yes, that's an excellent idea. The module is ✨ and a PR would be so, so welcome 🙇

**Comment by boneskull** [comment]
This may be much more involved than I had expected, but I'll take a look if I get a chance.
